pkg/api/addons.go

```diff
@@ -1131,3 +1171,12 @@ func kubeProxyImageSuffix(cs ContainerService) string {
 	}
 	return ""
 }
+
+func getCSISidecarComponent(csiDriverName, csiSidecarName string, k8sComponents map[string]string) string {
```

Comment:
I see this is being indirectly tested in the various cluster configs in `addons_test.go`, but would it make sense to add lightweight UT for just this func?